src/coreclr/jit/optcse.cpp

```diff
@@ -2219,7 +2219,7 @@ void CSE_HeuristicReplay::ConsiderCandidates()
         return;
     }
 
-    static ConfigIntArray JitReplayCSEArray;
+    ConfigIntArray JitReplayCSEArray;
```

Comment:
Do these functions get called multiple times per compilation? Now, `EnsureInit` will always re-build them.

Reply:
Yes, they do. 

I can make these into fields on the heuristic object so they only get set once per compilation. Currently I only expect they'll be read in "training" modes; the greedy policy will have baked-in settings.